sql/abilities.sql

```diff
@@ -270,53 +270,53 @@ INSERT INTO `abilities` VALUES (231,'composure',5,50,1,300,50,0,0,215,2000,0,6,2
+INSERT INTO `abilities` VALUES (341,'Rune_enchantment',22,5,1,0,92,0,0,0,2000,0,6,20.0,0,0,0,0,0,'SOA');
+INSERT INTO `abilities` VALUES (342,'Ignis',22,5,1,5,10,0,0,291,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (343,'Gelus',22,5,1,5,10,0,0,292,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (344,'Flabra',22,5,1,5,10,0,0,293,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (345,'Tellus',22,5,1,5,10,0,0,294,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (346,'Sulpor',22,5,1,5,10,0,0,295,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (347,'Unda',22,5,1,5,10,0,0,296,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (348,'Lux',22,5,1,5,10,0,0,297,2000,0,6,20.0,0,80,320,0,0,'SOA');
+INSERT INTO `abilities` VALUES (349,'Tenebrae',22,5,1,5,10,0,0,298,2000,0,6,20.0,0,80,320,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (350, 'Vallation',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,450,900,0,0,'SOA');
+INSERT INTO `abilities` VALUES (351,'Swordplay',22,20,1,300,0,0,0,299,2000,0,6,20.0,0,160,320,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (352,'Lunge',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,0,0,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (353,'Pflug',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,450,900,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (354,'Embolden',22,60,1,600,0,0,0,300,2000,0,6,20.0,0,160,320,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (355,'Valiance',22,50,1,300,0,0,0,???,2000,0,6,20.0,0,450,900,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (356,'Gambit',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,640,1280,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (357,'Liement',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,450,900,0,0,'SOA');
+INSERT INTO `abilities` VALUES (358,'One_for_all',22,95,1,300,0,0,0,301,2000,0,6,20.0,1,160,320,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (359,'Rayke',22,20,1,300,0,0,0,???,2000,0,6,20.0,0,640,1260,0,0,'SOA');
+-- INSERT INTO `abilities` VALUES (360,'Battuta',22,1,1,0,300,0,0,???,2000,0,6,20.0,0,450,900,0,0,'SOA');
+INSERT INTO `abilities` VALUES (363,'Ward',22,1,1,0,0,0,0,0,2000,0,6,20.0,0,0,0,0,0,NULL);
+INSERT INTO `abilities` VALUES (364,'Effusion',22,1,1,0,0,0,0,0,2000,0,6,20.0,0,0,0,0,0,NULL);
 INSERT INTO `abilities` VALUES (365,'chocobo_jig_ii',19,70,1,60,218,126,0,13,2000,0,14,20.0,1,1,300,0,0,'SOA');
-INSERT INTO `abilities` VALUES (367,'Vivacious_pulse',22,1,1,300,0,0,0,327,2000,0,6,20.0,0,1,300,0,0,NULL);
-INSERT INTO `abilities` VALUES (369,'apogee',15,70,1,180,108,0,0,333,2000,0,6,20.0,0,1,0,0,0,'SOA');
-INSERT INTO `abilities` VALUES (371,'bestial_loyalty',9,23,1,1200,94,0,0,83,2000,0,6,18.0,0,0,0,900,0,'SOA');
+INSERT INTO `abilities` VALUES (367,'Vivacious_pulse',22,1,1,300,0,0,0,327,2000,0,6,20.0,0,0,0,0,0,NULL);
+INSERT INTO `abilities` VALUES (369,'apogee',15,70,1,180,108,0,0,333,2000,0,6,20.0,0,1,80,0,0,'SOA');
```

Comment:
bgwiki is claiming 80 VE for Apogee as well

Reply:
• **Double shot**: although i have no numbers for CE and VE, reading the description and seeing how other damage CE and VE calculations are done i can conclude that this needs the majority of its hate calculated through it DMG delt in the lua and the base should be the same as other abilities that do not directly affect the mob, this could be wrong but it is an educated guess. 

• **Hagakure**: As this  does not directly affect the mob i concluded that the values should be lower, this is an on going trend with other abilites that do not directly affect the mob in any way and again could be wrong.

• **Bully**: Although this is higher now, it does not generate an amazing amount of hate, but it should be higher than it was due to the nature of the ability, again this is just an educated guess due to the trend of other abilities.

• **Stymie**: Again as this  does not directly affect the mob i concluded that the values should be lower, this is an on going trend with other abilites that do not directly affect the mob in any way and again could be wrong.

• **Mikage**: Again as this  does not directly affect the mob i concluded that the values should be lower, this is an on going trend with other abilites that do not directly affect the mob in any way and again could be wrong.

I am open to all opinions and changes relating to the values used and if anyone feels they are incorrect and wants to change any values and has reasons behind them, please feel to comment.